Combinatorial_map/include/CGAL/Combinatorial_map.h

```diff
@@ -40,6 +40,11 @@
 
 #include <CGAL/config.h>
 
+#if defined( __INTEL_COMPILER )
+#pragma warning (push)
+#pragma warning (disable:1017)
+#endif
+
```

Comment:
I suggest that you insert here a comment that describes the warning itself. Otherwise, those lines will be difficult to understand in a while.